aureliafile.js

```diff
@@ -0,0 +1,50 @@
+var cli = require('aurelia-cli');
+
+var config = {
+  js: {
```

Comment:
@EisenbergEffect We can also do `packagePath : 'path_of_package'` here. The default is `packagePath: '.'` which is appropriate for `skeleton-navigation`. Should we specify the default here so that people will know about the option?